### PR TITLE
Solomon places 6

### PR DESCRIPTION
**Pull request by sologebre**

### CODE REVIEW

File: 5001-6000/LOC5040Ogaden.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title>Ogaden</title>
+            <title>ʾOgaden</title>
             <author/>
```
Comment (DariaElagina) [inline 5001-6000/LOC5040Ogaden.xml]: gā

File: 5001-6000/LOC5040Ogaden.xml [hunk 2]
```diff
@@ -54,13 +54,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place type="region">
-               <placeName>Ogaden</placeName>
+            <place type="region" sameAs="Q137555">
+               <placeName xml:lang="gez" xml:id="n1">ኦጋዴን፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1" type="normalized">ʾOgaden</placeName>
                <placeName type="alt">Ogaadeen</placeName>
             </place>
```
Comment (DariaElagina) [inline 5001-6000/LOC5040Ogaden.xml]: gā

File: 5001-6000/LOC5040Ogaden.xml (hunk 1 above)
Comment (sologebre) [reply 5001-6000/LOC5040Ogaden.xml]: yes, corrected!

File: 2001-3000/LOC2952Enjiba.xml [hunk 3]
```diff
@@ -54,13 +54,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place>
-               <placeName>Ǝnǧibara</placeName>
+            <place type="town" sameAs="Q1110016">
+               <placeName xml:lang="gez" xml:id="n1">እንጅባራ፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1"  type="normalized">ʾƎnǧibārā</placeName>
             </place>
```
[the remark this answers is not in the document]
Comment (sologebre) [reply 2001-3000/LOC2952Enjiba.xml]: It should be እንጂባራ፡

File: 3001-4000/LOC3121Jaarso.xml [hunk 4]
```diff
@@ -54,13 +54,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place type="region">
-               <placeName>Ǧaarso</placeName>
+            <place type="region" sameAs="Q3162805">
+               <placeName xml:lang="gez" xml:id="n1">ጃርሶ፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1" type="normalized">Ǧāārso</placeName> 
             </place>
          </listPlace>
```
[the remark this answers is not in the document]
Comment (sologebre) [reply 3001-4000/LOC3121Jaarso.xml]: It is an Afaan Oromoo name

File: 1001-2000/LOC1579Asmara.xml [hunk 5]
```diff
@@ -54,13 +54,15 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPlace>
-            <place type="town">
-               <placeName>Asmära</placeName>
+            <place type="town" sameAs="Q3642">
+               <placeName xml:lang="gez" xml:id="n1">አስመራ፡</placeName>
+               <placeName xml:lang="gez" corresp="#n1" type="normalized">ʾAsmarā</placeName>
```
[the remark this answers is not in the document]
Comment (sologebre) [reply 1001-2000/LOC1579Asmara.xml]: The name Asmara may be originated from other language but the Gǝʿǝz form is አስመራ as correctly transcribed.

File: 2001-3000/LOC2952Enjiba.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (DariaElagina) [reply 2001-3000/LOC2952Enjiba.xml]: ok, then you should change it, now the transliteration does not correspond to fidal. And should it rather be Amharic? There is no ጂ in Geez.

File: 3001-4000/LOC3121Jaarso.xml (hunk 4 above)
[the remark this answers is not in the document]
Comment (DariaElagina) [reply 3001-4000/LOC3121Jaarso.xml]: Then it should be marked like that, now the transliteration does not correspond to the language. And, it is again not Geez

File: 1001-2000/LOC1579Asmara.xml (hunk 5 above)
[the remark this answers is not in the document]
Comment (DariaElagina) [reply 1001-2000/LOC1579Asmara.xml]: I am not sure that this is a geez form, it is written in fidal, but it is not automatically a geez word I think. The issue with the language seems to be a very complicated one.

File: 2001-3000/LOC2952Enjiba.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (sologebre) [reply 2001-3000/LOC2952Enjiba.xml]: First of all, we cannot say whether a place name is a "Gǝʿǝz language" or not, but only how this place name can be written in Fidal.  If we consider the etymological root of a place name, it becomes very complicated. We discussed this with Dorothea a few months ago. But when I say in the Gǝʿǝz form, it means that it is the way the place name is written in Gǝʿǝz literature or in Gǝʿǝz fidal. The place-name እንጂባራ appears in the Gǝʿǝz text like this.  That does not mean that it is exactly a Gǝʿǝz word. There are so many place names that we do not know for sure their etymological roots but appear in Gǝʿǝz texts. So we should write these places just in Gǝʿǝz fidal first, and then when we are sure we can add the transliteration system to the original language. In this case, for example, the word Ǧāārso  (Gǝʿǝz) or  Ǧāārso  (Oromifaa) is a place in the Oromiya region. I know Oromifaa that is why I said the Ǧāārso comes Oromifaa language which means "an elder person". However, there are several place names which come from the etymology of the languages we do not know yet- can be Agaw, Gafat, Gurage or other. The best thing, (I mean to be on the safe side ) is simply to write the place in Gǝʿǝz fidal and to transliterate it.

**Comment by PietroLiuzzo** [comment]
@DariaElagina I have resolved the conflicts for you. 
Please finalize the review, squash and merge, and then delete the branch as prompted.

**Pull request merged**